packages/web/pages/index.tsx

```diff
@@ -66,11 +48,36 @@ const Home = ({ ads }: InferGetStaticPropsType<typeof getStaticProps>) => {
       </div>
       <div className="my-auto flex h-auto w-full items-center">
         <div className="ml-auto mr-[15%] flex w-[27rem] flex-col gap-4 lg:mx-auto md:mt-mobile-header">
-          <SwapTool ads={ads} />
+          {featureFlags.swapsAdBanner && <SwapAdsBanner />}
+          <SwapTool />
         </div>
       </div>
     </main>
   );
 };
 
+const SwapAdsBanner = () => {
```

Comment:
just to confirm, the tanstack query will cache this call for us? otherwise, so it won't just fetch the banners on every load?

Remember, fe-content is a normal repo, meaning we will get rate limited (1k requests per hour if i remember correctly)

@JoseRFelix

Reply:
Yes, I will also add a local cache for 1 hour. Additionally, it's worth noting that this request is made entirely through the client and does not utilize our edge functions, so we won't face any rate-limiting issues